apps/condo/domains/ticket/schema/CreateResidentTicketService.js

```diff
@@ -19,9 +19,9 @@ const CreateResidentTicketService = new GQLCustomSchema('CreateResidentTicketSer
         {
             access: true,
             type:
-                'type ResidentTicketOutput { organization: Organization!, property: Property!, unitName: String,' +
-                'sectionName: String, floorName: String, number: Int!, client: User!, clientName: String,' +
-                'clientEmail: String, clientPhone: String, details: String!, related: Ticket, isEmergency: Boolean, status: TicketStatus!' +
+                'type ResidentTicketOutput { organization: Organization, property: Property, unitName: String,' +
```

Comment:
we always have organization it should be `!`. We always have property, number, details.

Reply:
done in 44b66c2